WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/ActivityIndicatorView.swift

```diff
@@ -0,0 +1,24 @@
+import SwiftUI
+
+/// This is a wrapper view for using the UIActivityIndicator in SwiftUI. The native progress indicator is not available under iOS 13.
+///
+struct ActivityIndicatorView: UIViewRepresentable {
+    @Binding var isAnimating: Bool
+    let style: UIActivityIndicatorView.Style
+
+    func makeUIView(context: UIViewRepresentableContext<ActivityIndicatorView>) -> UIActivityIndicatorView {
+        return UIActivityIndicatorView(style: style)
+    }
+
+    func updateUIView(_ uiView: UIActivityIndicatorView, context: UIViewRepresentableContext<ActivityIndicatorView>) {
+        isAnimating ? uiView.startAnimating() : uiView.stopAnimating()
+    }
+}
+
+struct ActivityIndicatorView_Previews: PreviewProvider {
```

Comment:
I always wonder, should this be under a `#if DEBUG` compiler flag? 
If not, this type would end up in our production binary, right?

Reply:
This is no more needed in Xcode 12, [Apple strips automatically the code under the PreviewProvider from the production binary](https://web.archive.org/web/20190911210340/https://developer.apple.com/documentation/xcode_release_notes/xcode_11_release_notes#3384944).
![aUkVp](https://user-images.githubusercontent.com/495617/115869922-d8f82200-a43e-11eb-8374-5c14de9ec6c5.png)

Reply:
Oh nice!